src/app/shared/navbar/navbar.scss

```diff
@@ -9,3 +11,7 @@
   height: 26px;
   vertical-align: middle;
 }
+
+.flex-spacer {
```

Comment:
does it make sense to prefix the classes if we're not prefixing the components? shouldn't we prefix everything or nothing